README.md

```diff
@@ -1,24 +1,175 @@
+### Tech Stack <a name="tech-stack"></a>
 
-* Services (job queues, cache servers, search engines, etc.)
+<details>
+  <summary>Ruby</summary>
+  <ul>
+<li>
 
-* Deployment instructions
+<a name="livelink"></a>
 
-* ...
+- [Click here to read more about ruby](https://www.ruby-lang.org/en/)
+
+   </li>
+  </ul>
+
+</details>
+
+<!-- Features -->
+
+### Key Features <a name="key-features"></a>
+
+- **[Add groups]**
+- **[Add Entries]**
+- **[Display groups and entries]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🚀 Live Demo <a name="live-demo"></a>
+
+- [Live Demo Link](https://mybudget-shzp.onrender.com)
+- [Video Link](https://www.loom.com/share/d467834ce70342349321fc1ef5eeb2ac?sid=9af5e4d7-67d7-4edc-9c4f-cd3e423a4b31)
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- GETTING STARTED -->
+
+## 💻 Getting Started <a name="getting-started"></a>
+
+To get a local copy up and running, follow these steps.
+
+### Prerequisites
+
+In order to run this project you need:
+
+```sh
+ bundle install
+```
+
+### Setup
+
+Clone this repository to your desired folder:
+
+```sh
+  cd Budget
+  git clone https://github.com/MayPyone/Budget-App.git
+```
+
+### Install
+
+Install this project with:
+
+```sh
+  cd Budget
+  bundle install
+  gem install --no-document rubocop -v '>= 1.0, < 2.0'
+  gem install rspec
+```
+
+### Usage
+
+To run the project, execute the following command:
+
+```sh
+  ruby main.rb
+  or
+  ./main
+```
+
+### Run tests
+
+To run tests, run the following command:
+
+- `rubocop --color`
+- `rspec spec --format doc`
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+### Deployment
+
+-
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+👤 **May Pyone**
+
+- GitHub: [@May Pyone](https://github.com/MayPyone/)
+- LinkedIn: [May Pyone](https://www.linkedin.com/in/may-pyone-9439961a3/)
+
+<!-- FUTURE FEATURES -->
+
+## 🔭 Future Features <a name="future-features"></a>
+
+- [ ] **[Delete each entry]**
+- [ ] **[Delete each group]**
+- [ ] **[Edit profile]**
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- CONTRIBUTING -->
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- SUPPORT -->
+
+## ⭐️ Show your support <a name="support"></a>
+
+If you like this project please show support by staring ⭐️.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- I would like to thank I would like to thank [Gregoire Vella](https://www.behance.net/gregoirevella) for this beautiful design and Microverse team.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] The license link in the README file is broken, Please provide a valid link.

![image](https://github.com/MayPyone/Budget/assets/68796232/62ec8cf5-b934-4e5c-8829-8aa2107b7fe2)